honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,327 @@
+
+    exterior_modifier: str = Field(
+            default=None,
+            description='Identifier for a radiance modifier object for faces with an '
+                        ' Outdoors boundary condition.'
+        )
+
+    interior_modifier: str = Field(
+            default=None,
+            description='Identifier for a radiance modifier object for faces with a '
+                        'boundary condition other than Outdoors.'
+        )
+
+
+class WallModifierSetAbridged(BaseModifierSetAbridged):
+    """Abridged set containing radiance modifiers needed for a model's Walls."""
+
+    type: constr(regex='^WallModifierSetAbridged$') = 'WallModifierSetAbridged'
+
+
+class FloorModifierSetAbridged(BaseModifierSetAbridged):
+    """Abridged set containing radiance modifiers needed for a model's Floors."""
+
+    type: constr(regex='^FloorModifierSetAbridged$') = 'FloorModifierSetAbridged'
+
+
+class RoofCeilingModifierSetAbridged(BaseModifierSetAbridged):
+    """Abridged set containing radiance modifiers needed for a model's Roofs."""
+
+    type: constr(regex='^RoofCeilingModifierSetAbridged$') = \
+        'RoofCeilingModifierSetAbridged'
+
+
+class ShadeModifierSetAbridged(BaseModifierSetAbridged):
+    """Abridged set containing radiance modifiers needed for a model's Shade."""
+
+    type: constr(regex='^ShadeModifierSetAbridged$') = 'ShadeModifierSetAbridged'
+
+
+class ApertureModifierSetAbridged(NoExtraBaseModel):
+    """Abridged set containing radiance modifiers needed for a model's Apertures."""
+
+    type: constr(regex='^ApertureModifierSetAbridged$') = 'ApertureModifierSetAbridged'
+
+    window_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, '
+                        'and Wall parent Face.'
+        )
+
+    interior_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with a Surface '
+                        'boundary condition.'
+        )
+
+    skylight_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, and a '
+                        'RoofCeiling or Floor face type for their parent face.'
+        )
+
+    operable_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with an Outdoors '
+                        'boundary condition and a True is_operable property.'
+        )
+
+
+class DoorModifierSetAbridged(BaseModifierSetAbridged):
+    """Abridged set containing radiance modifiers needed for a model's Doors."""
+
+    type: constr(regex='^DoorModifierSetAbridged$') = 'DoorModifierSetAbridged'
+
+    exterior_modifier: str = Field(
```

Comment:
This `exterior_modifier` property is the same between `BaseModifierSetAbridged` and `DoorModifierSetAbridged`. So I think you can delete it.